administrator/components/com_languages/tmpl/languages/default.php

```diff
@@ -63,9 +63,15 @@
 									<?php echo JHtml::_('searchtools.sort', 'COM_LANGUAGES_HEADING_LANG_IMAGE', 'a.image', $listDirn, $listOrder); ?>
 								</th>
 								<th style="width:10%" class="nowrap d-none d-md-table-cell text-center">
+									<?php echo JHtml::_(
+										'searchtools.sort', 'COM_LANGUAGES_HEADING_AUTOMATIC_ASSOCIATION', 'a.automatic_association', $listDirn,
+										$listOrder
+									); ?>
+								</th>
+								<th style="width:5%" class="nowrap d-none d-md-table-cell text-center">
 									<?php echo JHtml::_('searchtools.sort', 'JGRID_HEADING_ACCESS', 'a.access', $listDirn, $listOrder); ?>
 								</th>
-								<th style="width:10%" class="nowrap d-none d-md-table-cell text-center">
+								<th style="width:5%" class="nowrap d-none d-md-table-cell text-center">
```

Comment:
I suggest to remove the class .nowrap from all <th> tags. Nobreak enforces the table header to be too big and causes an overflow

Reply:
I also don't like inline `style=""` elements

Reply:
The width attribute here is ugly, but this is general use in tables everywhere, so I think we must not change this.

Reply:
:+1: